### PR TITLE
Added new tab style and changed some colors

### PR DESCRIPTION
**Pull request by AlexeyGreer**
- Added new SegmentedTabs style
- Changed some colors

### CODE REVIEW

File: src/foam/u2/tag/Button.js [hunk 1]
```diff
@@ -203,8 +203,8 @@ foam.CLASS({
 
     ^secondary-destructive:disabled {
       background-color: $buttonSecondaryColor$hover;
```
Comment (sarthak-marwaha) [inline src/foam/u2/tag/Button.js:205]: Can we also change the colour on L:201 to destrutive500

File: src/foam/u2/Tabs.js [hunk 2]
```diff
@@ -91,3 +91,70 @@ foam.CLASS({
...
+      flex: 1;
+      position: relative;
+    }
+    ^tabRow {
+      flex: 0 1 auto;
+      border-radius: 3px;
+      border: 1px solid $grey100;
+      display: flex;
+      gap: 8px;
+      padding: 3px;
```
Comment (sarthak-marwaha) [inline src/foam/u2/Tabs.js]:
```suggestion
      padding: 0.4rem;
```

**Comment by kgrgreer** [comment]
LGTM

File: src/foam/u2/tag/Button.js [hunk 3]
```diff
@@ -198,13 +198,13 @@ foam.CLASS({
 
     ^secondary-destructive:active:not(:disabled) {
       background-color: $buttonSecondaryColor$hover;
-      border-color: $red500;
+      border-color: $$destructive500;
```
Comment (sarthak-marwaha) [inline src/foam/u2/tag/Button.js]:
```suggestion
      border-color: $destructive500;
```

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**